controller/lib/src/adp.h

```diff
@@ -82,6 +82,13 @@ class adp
     ///
     /// Get the Controller Entity ID of the AVDECC Entity sending the command.
     ///
+    /// This function is used by descriptor commands to fetch the
+    /// controller entity id corresponding to a particular end station.
```

Comment:
This text does not make sense. An end station does not have a controller entity id.